apigentools/commands/generate.py

```diff
@@ -387,7 +387,17 @@ def pull_repository(self, language, branch=None):
                 )
                 try:
                     self.setup_git_config(cwd=output_dir)
-                    run_command(["git", "merge", self.args.is_ancestor], cwd=output_dir)
+                    run_command(
+                        [
+                            "git",
+                            "merge",
+                            "-Xtheirs",
```

Comment:
I wouldn't automatically resolve conflicts when they are some, as it could cause unexpected loss in one's work

Reply:
without this it's useless. i have tried

Reply:
if there is an actual conflict it will still fail

Reply:
Hmm, I actually use that when I don't care about conflicts, and want to resolve everything automatically by accepting the others change. Pretty sure it doesn't fail in case of real conflicts